apps/nextra/pages/en/build/guides/exchanges.mdx

```diff
@@ -0,0 +1,1180 @@
+    "authentication_key": "0xa746e980ae21949a4f084db7403430f00bce3c9a1da4101ffcf0bf45ebd35e7e",
+    "coin_register_events": {
+    "counter": "14",
+    "guid": {
+    "id": {
+    "addr": "0xa746e980ae21949a4f084db7403430f00bce3c9a1da4101ffcf0bf45ebd35e7e",
+    "creation_num": "0"
+  }
+  }
+  },
+    "guid_creation_num": "44",
+    "key_rotation_events": {
+    "counter": "0",
+    "guid": {
+    "id": {
+    "addr": "0xa746e980ae21949a4f084db7403430f00bce3c9a1da4101ffcf0bf45ebd35e7e",
+    "creation_num": "1"
+  }
+  }
+  },
+    "rotation_capability_offer": {
+    "for": {
+    "vec": []
+  }
+  },
+    "sequence_number": "52",
+    "signer_capability_offer": {
+    "for": {
+    "vec": []
+  }
+  }
+  }
+  },
+    "type": "write_resource"
+  }
+    ],
+    "sender": "0xa746e980ae21949a4f084db7403430f00bce3c9a1da4101ffcf0bf45ebd35e7e",
+    "sequence_number": "51",
+    "max_gas_amount": "817",
+    "gas_unit_price": "100",
+    "expiration_timestamp_secs": "1724196316",
+    "payload": {
+    "function": "0x1::primary_fungible_store::transfer",
+    "type_arguments": [
+    "0x1::fungible_asset::Metadata"
+    ],
+    "arguments": [
+  {
+    "inner": "0xa"
+  },
+    "0x5",
+    "10000"
+    ],
+    "type": "entry_function_payload"
+  },
+    "signature": {
+    "public_key": "0x330e75a102e37270b788caee8dd819e5badedd5fa17fe9f72017732e9bb98c60",
+    "signature": "0xd4666df2887cf2d8192230e4a03d842ea75a86ffbc46a9a16a9baede6ff646c6b2bcafc524d3a4a7a66c223b5db576beb5cfefbd549620e69097c0a364c7a800",
+    "type": "ed25519_signature"
+  },
+    "events": [
+  {
+    "guid": {
+    "creation_number": "0",
+    "account_address": "0x0"
+  },
+    "sequence_number": "0",
+    "type": "0x1::fungible_asset::Withdraw",
+    "data": {
+    "amount": "10000",
+    "store": "0x7ed92ce166e251fc133f6b4d46a6b41307962e3b6864c2231110b3808648188"
+  }
+  },
+  {
+    "guid": {
+    "creation_number": "0",
+    "account_address": "0x0"
+  },
+    "sequence_number": "0",
+    "type": "0x1::fungible_asset::Deposit",
+    "data": {
+    "amount": "10000",
+    "store": "0x8a4613c356c21a45045e06dcc404bfee363aabd65a774d4d43defd71289239b2"
+  }
+  },
+  {
+    "guid": {
+    "creation_number": "0",
+    "account_address": "0x0"
+  },
+    "sequence_number": "0",
+    "type": "0x1::fungible_asset::Withdraw",
+    "data": {
+    "amount": "54500",
+    "store": "0x7ed92ce166e251fc133f6b4d46a6b41307962e3b6864c2231110b3808648188"
+  }
+  },
+  {
+    "guid": {
+    "creation_number": "0",
+    "account_address": "0x0"
+  },
+    "sequence_number": "0",
+    "type": "0x1::transaction_fee::FeeStatement",
+    "data": {
+    "execution_gas_units": "6",
+    "io_gas_units": "7",
+    "storage_fee_octas": "53240",
+    "storage_fee_refund_octas": "0",
+    "total_charge_gas_units": "545"
+  }
+  }
+    ],
+    "timestamp": "1724196287102837",
+    "type": "user_transaction"
+  }
+  ```
+</details>
+
+## Transferring Assets
+
+### Coin (or migrated coin) Transfers
```

Comment:
could we highlight you must use coin::transfer for migrated coins and pfs::transfer to pure FA.
And coin are all migrated so there is no `Coin`, only `Migrated Coin`.